test/testdata/lsp/workspace_symbols_stdlib.rb

```diff
@@ -6,7 +6,7 @@
 # symbol-search: "conjugate", container="::Numeric", uri="numeric.rbi"
 
 # // Multiple symbols from same rbi file?
-# symbol-search: "encoding", name="encoding", container="::Regexp", uri="regexp.rbi"
+# symbol-search: "encoding", name="::Regexp#encoding", container="::Regexp", uri="regexp.rbi"
 
 # // Extremely common
-# symbol-search: "hash", name="hash", container="::Kernel", uri="kernel.rbi"
+# symbol-search: "hash", name="::Kernel#hash", container="::Kernel", uri="kernel.rbi"
```

Comment:
🤔  what's with the `#`? Does this indicate a field?

Reply:
we discussed; `#` is standard for fields in rails and is probably fine.

Reply:
In ruby docs (for some reason I have no idea), the convention is `Class#method`

Reply:
`Class#instance_method` and `Class.class_method` are different syntax to separate instance and class methods in Ruby.

Reply:
This won't work for class methods currently as Sorbet cannot print classes (it would show up as `<Foo Class>`) well so at least this won't be worse than current:
https://ruby-doc.org/documentation-guidelines.html

 I can hopefully do a followup to make this work for Class methods with nice formatting there

Reply:
> I can hopefully do a followup to make this work for Class methods with nice formatting there

Please do! Do we currently format them with `<Foo Class>` in search results? D: